content/calico/stars_policy_demo/directional_traffic.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Allow Directional Traffic"
+date: 2018-08-07T08:30:11-07:00
+weight: 4
+---
+Let's see how we can allow directional traffic from client to frontend and backend.
+
+To allow traffic from frontend service to the backend service apply the following manifest:
+
+```
+kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/tutorials/stars-policy/policies/backend-policy.yaml
+```
+
+And allow traffic from the client namespace to the frontend service:
+
+```
+kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/tutorials/stars-policy/policies/frontend-policy.yaml
+```
+Upon refreshing your browser, you should be able to see the network policies in action:
+
+![directional traffic](/images/calico-client-f-b-access.png)
+
+Let's have a look at the backend-policy. Its spec has a podSelector that selects all pods with the label **role:backend**, and allows ingress from all pods that have the label **role:frontend** and on TCP port **6379**
```

Comment:
I would also add that the traffic is allowed in one direction and specific based on port number (6379 in this case)